### ISSUE TITLE
loops: monster segment name issue.

### ISSUE THREAD

**Issue by woutershep** [omsi6/omsi6.github.io#38]
**Describe the bug**
Fight monster segment modifier is wrong. It states some internal identifier.

**To Reproduce**
1. Go fight monsters.
2. Hover over the segment.
3. See the problem

**Expected behavior**
Instead of a internal identifier i expect to see the proper segment modifier.

**Screenshots**
![2019-05-11-134704_1920x1080_scrot](https://user-images.githubusercontent.com/11578180/57569348-0aaf8400-73f4-11e9-9796-33cdc821cef8.png)

**Savefile**
Can provide if needed.

**Comment by omsi6** [comment]
I can't seem to immediately reproduce it, if you could provide your savefile, that would be appreciated.

**Comment by woutershep** [comment]
See attached file for save game. [loops2.txt](https://github.com/omsi6/omsi6.github.io/files/3170234/loops2.txt)

However i have some additional info, I think it has been a problem with cached files. If i hovered over Changelog it would show 0.92. I have loaded the page last Friday, 10 May, so well after the last change of 5 May. However i have just done a reload ignoring cache and suddenly the screen looks different with zone colors and the fontsize of the buffs are different. In addition I can no longer reproduce this issue. I noticed you are versioning the js files so i don't fully understand how this could have happened. Maybe the visual changes was a cached stylesheet.css, it seems to lack the `?version` tag so that part is explainable at least. I think maybe this issue was caused by a cached game.xml? But i don't understand the localization setup fully so i don't know if that's possible, i'm just guessing.

Long story short, i can no longer reproduce after reload without cache, feel free to close the issue.

**Comment by omsi6** [comment]
Yeah I don't fully understand the cache stuff, except that a: it happens sometimes, and b: I can't seem to ever get it to happen to me.
